kernel/indTyping.ml

```diff
@@ -67,7 +67,7 @@ let mind_check_names mie =
 type univ_info = { ind_squashed : bool; ind_has_relevant_arg : bool;
                    ind_min_univ : Sorts.t option; (* Some for template *)
                    ind_univ : Sorts.t;
-                   missing : Universe.Set.t; (* missing u <= ind_univ constraints *)
+                   missing : Sorts.t list; (* missing u <= ind_univ constraints *)
```

Comment:
Shouldn't we uniquize at some point?

Reply:
Doesn't matter, this is only used for printing and I am not even sure we can really generate redundant constraints from user-facing code.

Reply:
It being used for printing is why it's nicer to uniquize IMO

Reply:
Even that way, it's an error that can only be raised when the upper layers are buggy. I don't think we exercise this code path currently.

Reply:
We probably will need Sort sets anyway?

Reply:
Maybe, but introducing it here is a bit of premature overengineering in my opinion...